docs/docs/howto/workload-deployment/deployment-file-preparation.md

```diff
@@ -68,6 +68,27 @@ This is a placeholder name that will be replaced by a versioned `runtimeClassNam
 
 <!-- TODO(katexochen): Describe how runtimeClass is handled after first generate -->
 
+### Volumes
+
+Contrast doesn't support sharing filesystems with the host.
+This means that most [Kubernetes volume](https://kubernetes.io/docs/concepts/storage/volumes/) mounts aren't supported.
+Depending on the type, an unsupported volume mount in the pod spec leads to startup errors, ephemeral mounts or copying of files between the host and the guest.
+
+The easiest and safest way to mount a persistent filesystem into a container is Contrast's [secure persistent volume](../../architecture/secrets.md#secure-persistence) support.
```

Comment:
Not sure this should link to the architecture section or to the how-to encrypted-storage?

Reply:
Yeah, howto makes more sense - changed.